<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoPlayer</class>
 <widget class="QMainWindow" name="VideoPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="layout">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <item>
       <widget class="QWidget" name="video_widget" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="progress_widget" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="label_startTime">
           <property name="text">
            <string>00:00:00</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="Slider_progress">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_endTime">
           <property name="text">
            <string>00:00:00</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="set_widget" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>55</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="btn_play">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
color:white;
border-radius:25px;
border:2px groove gray;
border-style:outset;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_stop">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
color:white;
border-radius:25px;
border:2px groove gray;
border-style:outset;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="btn_open">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
color:white;
border-radius:20px;
border:2px groove gray;
border-style:outset;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_speed">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <item>
              <property name="text">
               <string>X0.5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>X0.75</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>X1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>X1.25</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>X1.5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>X1.75</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>X2</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>206</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSlider" name="Slider_volume">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>20</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1032</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuhelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionhelp2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuhelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionhelp2">
   <property name="text">
    <string>介绍</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
